vertx/feign-vertx/src/main/java/feign/VertxFeign.java

```diff
@@ -263,34 +261,6 @@ public Builder errorDecoder(final ErrorDecoder errorDecoder) {
       return this;
     }
 
-    /**
-     * Sets request options using Vert.x {@link HttpClientOptions}.
-     *
-     * @param options {@code HttpClientOptions} for full customization of the underlying Vert.x
-     *     {@link HttpClient}
-     * @return this builder
-     */
-    public Builder options(final HttpClientOptions options) {
-      this.options = checkNotNull(options, "Argument options must be not null");
-      return this;
-    }
-
-    /**
-     * Sets request options using Feign {@link Request.Options}.
-     *
-     * @param options Feign {@code Request.Options} object
-     * @return this builder
-     */
-    @Override
-    public Builder options(final Request.Options options) {
-      checkNotNull(options, "Argument options must be not null");
-      this.options =
-          new HttpClientOptions()
-              .setConnectTimeout(options.connectTimeoutMillis())
-              .setIdleTimeout(options.readTimeoutMillis());
-      return this;
-    }
```

Comment:
Same as above, is it possible to keep as deprecated!?

Reply:
Idem. Current version from core project was never released.